### DISCUSSION TITLE
Data una proforma, come scaricare il suo DDT .pdf collegato?

### DISCUSSION THREAD

**Discussion by dmigliacci** [fattureincloud/api#186] [Q&A] [answered]
Buongiorno,
sul nostro applicativo creiamo proforme con corrispondente DDT.

Dato l'id della proforma, c'è un modo per reperire l'indirizzo pdf del DDT che si è generato?

Grazie in anticipo,
Davide

**Comment by emajo** [comment, marked as answer]
Ciao @dmigliacci, si, facendo una chiamata [Get Issued Document](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents/-document_id-) viene restituito il campo `dn_url` che contiene l'url del pdf del ddt allegato.

Effettivamente non lo vedo documentato, grazie per la segnalazione,
Emanuele.

**Reply by dmigliacci** [reply]
Buongiorno e grazie per la risposta, con la get intendi la get del documento di trasporto (ma dove prendo l'id?) oppure la get della proforma generatrice (di cui ho l'id)?

**Reply by emajo** [reply]
La get della proforma.

**Reply by dmigliacci** [reply]
Ah ok ottimo, allora dovrei già avere il tutto, solo non sapevo della proprietà dn_url.

Controllo e ti aggiorno,

Thx

**Reply by dmigliacci** [reply]
No mi spiace, a quanto pare non è incluso l'url del DN.

![Screenshot 2022-12-09 alle 11 33 24](https://user-images.githubusercontent.com/43272717/206684036-239d52ab-b0d1-4a67-b1b4-b35d23930f6a.png)

**Reply by emajo** [reply]
Ciao @dmigliacci le SDKs richiedono la modifica lato nostro, crediamo di riuscire a rilasciare una nuova versione già nei prossimi giorni.

Grazie,
Emanuele.

**Reply by dmigliacci** [reply]
Ciao Emanuele,
Si scusa avevo capito mancasse solo la documentazione e ho fatto una verifica sul codice.

Non c'è problema, grazie a presto.

**Reply by emajo** [reply]
Ciao @dmigliacci, abbiamo rilasciato una nuova versione delle SDKs dove troverai il campo dn_url, facci sapere se abbiamo risolto il tuo problema,

grazie,
Emanuele.

**Reply by dmigliacci** [reply]
Ciao @emajo ,
per caso nell'ultimo rilascio avete rinominato nell'oggetto **IssuedDocumentItemsListItem** la funzione **inDdt** in **inDn**??

**Reply by emajo** [reply]
Ciao @dmigliacci, si è stato rinominato quel campo sulle sdk in quanto inDdt non è mai esistito per le nostre api ma si chiama appunto inDn.
